_topic_maps/_topic_map.yml

```diff
@@ -3023,10 +3023,10 @@ Topics:
   File: virt-learn-more-about-openshift-virtualization
   Distros: openshift-origin
 - Name: OpenShift Virtualization release notes
-  File: virt-4-8-release-notes
+  File: virt-4-11-release-notes
```

Comment:
I don't think you need to edit the `_topic_map.yml` file as part of this PR. Only the one dummy `virt-4-11-release-notes.adoc` file should be added.